tl/features/cell_context_matches.py

```diff
@@ -213,16 +218,97 @@ def initialize(self, input_df, context_dict, label_column):
                                            score=context_result['score']
                                            )
 
+        property_val_df, important_properties = self.compute_property_scores(row_col_label_dict, n_cntxt_columns, num_rows)
```

Comment:
lines 221 - 247 , put them in a new function

Reply:
does this needs to be in the initialize function?

Reply:
completed.